src/main/controllers/RespondentAddressController.ts

```diff
@@ -88,16 +89,28 @@ export default class RespondentAddressController {
 
   public get = (req: AppRequest, res: Response): void => {
     const redirectUrl = setUrlLanguage(req, PageUrls.RESPONDENT_ADDRESS);
+    const userCase = req.session.userCase;
 
     const content = getPageContent(req, this.respondentAddressContent, [
       TranslationKeys.COMMON,
       TranslationKeys.RESPONDENT_ADDRESS,
       TranslationKeys.SIDEBAR_CONTACT_US,
     ]);
-    assignFormData(req.session.userCase, this.form.getFormFields());
+
+    const respondentAddress = answersAddressFormatter(
+      userCase.respondents[0].respondentAddress.AddressLine1,
+      userCase.respondents[0].respondentAddress.AddressLine2,
+      userCase.respondents[0].respondentAddress.AddressLine3,
+      userCase.respondents[0].respondentAddress.PostTown,
+      userCase.respondents[0].respondentAddress.County,
+      userCase.respondents[0].respondentAddress.Country,
+      userCase.respondents[0].respondentAddress.PostCode
```

Comment:
This might be my lack of knowledge with typescript and how things are done in et-syr but this seems like it's only looking at the first respondent. In a case with multiple respondents, would this show the first respondent's address if you are the second respondent on the case?

Reply:
Updated to use userCase.respondentAddressLine1, userCase.respondentAddressLine2 etc etc.